runhouse/utils.py

```diff
@@ -800,3 +810,335 @@ def print_sky_clusters_msg(num_sky_clusters: int):
         console.print(
             f"{msg} For more information, please run [bold italic]sky status -r[/bold italic]."
         )
+
+
+def adjust_resource_type(resource_type: str):
+    """
+    status helping function. transforms a str form runhouse.resources.{X.Y...}.resource_type to runhouse.resource_type
+    """
+    try:
+        resource_type = resource_type.split(".")[-1]
+        getattr(importlib.import_module("runhouse"), resource_type)
+        return f"runhouse.{resource_type}"
+    except AttributeError:
+        return resource_type
+
+
+def get_resource_link_in_den_ui(cluster_name: str, api_server_url: str):
+    from runhouse.globals import rns_client
+
+    cluster_uri = rns_client.format_rns_address(cluster_name)
+    link_to_den_dashboard = f"{api_server_url}/resources/{cluster_uri}"
+    return link_to_den_dashboard
+
+
+###############################
+# Status helping functions
+###############################
+
+
+def print_cluster_config(cluster_config: Dict):
```

Comment:
these console printing helpers are more specifically cli utils, maybe we move those into a dedicated `cli_utils` file? (specifcally the ones that are only relevant to the CLI)